content/guides/end-to-end-testing/auth0-authentication.md

```diff
@@ -111,10 +117,146 @@ require('dotenv').config()
 }
 ```
 
+Note that `auth0_client_secret` is only needed for
+[programmatic login](#Programmatic-Login).
```

Comment:
I don't think the scope or audience should be needed as well, right?

Reply:
They're not needed specifically for our tests, they're needed for integrating Auth0 into an app. A lot of stuff on this page is just general Auth0 setup guidance (which I think is something we should review) so I felt it should be left in for now.